sanic/pages/styles/BasePage.css

```diff
@@ -2,32 +2,40 @@
 
 :root {
     --sanic: #ff0d68;
-    --sanic-blue: #0092FF;
     --sanic-yellow: #FFE900;
-    --sanic-purple: #833FE3;
-    --sanic-green: #37ae6f;
-    --sanic-background: #f1f5f9;
-    --sanic-text: #1f2937;
-    --sanic-tab-background: #fff;
-    --sanic-tab-text: #0f172a;
-    --sanic-tab-shadow: #adadad;
+    --sanic-background: #fff;
```

Comment:
I would suggest either a slightly darker white or even some color here. A combination of 000 and fff is too highly contrasted on modern displays, and fff should really only be used for highlights. I do realize that a lot of web design is still based on those default colors but it really helps to eyes and makes text more readable if there is a little less brightness. Using colors other than shades of gray would be also nice for the overall look of web but it is harder to pick color combinations that work well.

Reply:
![image](https://user-images.githubusercontent.com/166269/223098671-8b6f7dc7-8f9c-4ed6-a886-92d887220e3d.png)
How about like this?

Reply:
Looks better. Probably also should use inherited/same background on details list, and remove padding of the wrapping div so that it aligns with the rest (now it still seems to use fff background and is slightly indented compared to the rest of the page, and in particular to other h2 headings that all should look the same and be aligned the same.

Reply:
Fixed. All contents now aligned.